src/pages/deploy/config-as-code.md

```diff
@@ -0,0 +1,199 @@
+---
+title: Config as Code
+---
+
+<PriorityBoardingBanner />
+
+Railway supports defining the configuration for a single deployment in a file alongside your code. By default, we will look for a `railway.toml` or `railway.json` file. Everything in the build and deploy sections of the service settings page can be specified in this configuration file.
+
+_This is a beta feature and may change in the near future. Use at your own risk._
+
+## Examples
+
+These configuration files are equivalent.
```

Comment:
nit: Does providing the following `toml` produce that JSON output. This section header is a little confusing.

Reply:
Yup exactly. Both formats are the same